feature-settings/src/main/java/cm/aptoide/pt/settings/presentation/SettingsView.kt

```diff
@@ -253,14 +254,20 @@ fun SettingsScreen(
       )
 
       LongDivider()
-      Header(title = "Notifications")
+      Header(title = "Storage")
       ExtraOption(
+        modifier = Modifier
```

Comment:
This exact modifier should be inside the composable, instead of being declared multiple times in each instance